docs/DeveloperGuide.md

```diff
@@ -1039,44 +1039,74 @@ testers are expected to do more *exploratory* testing.
 
 </div>
 
-### Launch and shutdown
-
-1. Initial launch
-
-   1. Download the jar file and copy into an empty folder
-
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
-
-1. Saving window preferences
-
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
-
-   1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
-
-1. _{ more test cases …​ }_
-
-### Deleting a person
-
-1. Deleting a person while all persons are being shown
-
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
-
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
-
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
-
-1. _{ more test cases …​ }_
-
-### Saving data
-
-1. Dealing with missing/corrupted data files
-
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
-
-1. _{ more test cases …​ }_
+### Recipe
+
+1. Deleting a recipe ingredient
+    1. Prerequisites: Recipe at index 1 has at least 1 ingredient and at most 3 ingredients.
+    1. Test case: `delete-ri 1 i/1`
+       <br>Expected: First ingredient is deleted from the ingredients in the first recipe of the displayed recipe bookmarks list.
+       Details of the deleted ingredient and edited recipe is shown in the result display box.
+    1. Test case: `delete-ri 1 i/10`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri abc i/1`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri 1 i/abc`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri 1`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri i/1`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+1. Editing a recipe ingredient
+    1. Prerequisites: Recipe bookmarks list shows at least 1 recipe and at most 3 recipes.
```

Comment:
Need to include that there's no existing duplicate recipe? And test case failure when trying to edit a recipe when it already exists
```suggestion
    1. Prerequisites: Recipe bookmarks list only has these 3 recipes:
        * name: Pecan pie, price: $10
        * name: Banana split, price: $10
        * name: Orange cake, price: $12
```

Reply:
Oh that's true. I think it's better to not include those cases because it becomes additional work to explain.

Reply:
Ok that's fine. Anyways this is just a guide, it doesn't need to be exhaustive.